cli/cdk/spinner.go

```diff
@@ -0,0 +1,12 @@
+package cdk
+
+import "github.com/odpf/salt/printer"
+
+func WithSpinner(fn func(done func()) error) error {
+	spinner := printer.Spin("")
+	defer spinner.Stop()
```

Comment:
Wouldn't this stop the spinner as soon as `WithSpinner` call finishes, which might be earlier than expected?